reference/commands/creator/export-pkg.rst

```diff
@@ -60,42 +62,57 @@ and or '--profile'.
                             be written
 
 
-:command:`conan export-pkg` executes the following methods of a *conanfile.py* whenever ``--package-folder`` is used:
+The :command:`export-pkg` command let you create a package from already existing files
+in your working folder, it can be useful if you are using a build process external to Conan
+and do not want to provide it with the recipe. Nevertheless, you should take into
+account that it will generate a package and Conan won't be able to guarantee its
+reproducibility or regenerate it again. This is **not** the normal or recommended flow
+for creating Conan packages.
+
+Execution of this command will result in several files copied to the package
+folder in the cache identified by its ``package_id`` (Conan will perform all the
+required actions to compute this _id_: build the graph, evaluate the requirements and
+options, and call any required method), but there could be two
+different sources for the files:
 
-1. ``config_options()``
-2. ``configure()``
-3. ``requirements()``
-4. ``package_id()``
+ * If the argument ``--package-folder`` is provided, Conan will just copy all the
+   contents of that folder to the package one in the cache.
+ * If no ``--package-folder`` is given, Conan will execute the method ``package()``
+   of the recipe twice: first using the ``source_folder`` and then ``build_folder``
```

Comment:
This is no longer true, only 1 call to ``package()`` is done, and ``FileCopier`` is able to internally iterate the 2 folders (source and build)

Reply:
I'll change it in another PR right now

Reply:
Here: https://github.com/conan-io/docs/pull/1304